tests/cli/test_flow_run.py

```diff
@@ -221,15 +221,15 @@ class TestCancelFlowRun:
     @pytest.mark.parametrize(
         "state",
         [
-            Scheduled,
             Running,
-            Late,
             Pending,
-            AwaitingRetry,
```

Comment:
As above, should `AwaitingRetry` named states still be put in a `Cancelling` state?

Reply:
As long as they don't have an infra PID